integrations/intercom/src/index.ts

```diff
@@ -19,13 +19,12 @@ const log = console
 const integrationName = 'intercom'
 
 const conversationPartSchema = z.object({
-  type: z.literal('conversation_part'),
   id: z.string(),
   author: z.object({
     id: z.string(),
     type: z.string(),
   }),
-  body: z.string(),
+  body: z.string().nullable(),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Is this really nullable? I don't seem to read anywhere is the docs that this is the case?

Reply:
@ptrckbp is it undefined or null? because there is a difference between `.optionnal()` and `.nullable()`

Reply:
yeah, I remembered wrong, I used optional before but it was null, so nullable is the right thing. I just reconfirmed. 
This is an example req.body showcasing it: 
{
  "type": "notification_event",
  "app_id": "ouy9scuc",
  "data": {
    "type": "notification_event_data",
    "item": {
      "type": "conversation",
      "id": "47",
      "created_at": 1688053573,
      "updated_at": 1688053574,
      "waiting_since": 1688053574,
      "snoozed_until": null,
      "source": {
        "type": "conversation",
        "id": "993736671",
        "delivered_as": "customer_initiated",
        "subject": "",
        "body": "<p>zoom zoom zoom</p>",
        "author": {
          "type": "lead",
          "id": "710306658bjds4ckd34257f9",
          "name": null,
          "email": ""
        },
        "attachments": [],
        "url": "https://app.intercom.com/hosted_messenger/ouy9scuc?__inter-m5",
        "redacted": false
      },
      "contacts": {
        "type": "contact.list",
        "contacts": [{ "type": "contact", "id": "710306658bjds4ckd34257f9" }]
      },
      "first_contact_reply": {
        "created_at": 1688053573,
        "type": "conversation",
        "url": "https://app.intercom.com/hosted_messenger/ouy9scuc?__inter-m5"
      },
      "admin_assignee_id": 4977135,
      "team_assignee_id": null,
      "open": true,
      "state": "open",
      "read": true,
      "tags": { "type": "tag.list", "tags": [] },
      "priority": "not_priority",
      "sla_applied": null,
      "statistics": {
        "type": "conversation_statistics",
        "time_to_assignment": 1,
        "time_to_admin_reply": null,
        "time_to_first_close": null,
        "time_to_last_close": null,
        "median_time_to_reply": null,
        "first_contact_reply_at": "2023-06-29T15:46:13.000Z",
        "first_assignment_at": "2023-06-29T15:46:14.000Z",
        "first_admin_reply_at": null,
        "first_close_at": null,
        "last_assignment_at": "2023-06-29T15:46:14.000Z",
        "last_assignment_admin_reply_at": null,
        "last_contact_reply_at": "2023-06-29T15:46:13.000Z",
        "last_admin_reply_at": null,
        "last_close_at": null,
        "last_closed_by_id": null,
        "count_reopens": 0,
        "count_assignments": 1,
        "count_conversation_parts": 2
      },
      "conversation_rating": null,
      "teammates": { "type": "admin.list", "admins": [] },
      "title": "",
      "custom_attributes": {},
      "topics": { "type": "topic.list", "topics": [], "total_count": 0 },
      "ticket": null,
      "conversation_parts": {
        "type": "conversation_part.list",
        "conversation_parts": [
          {
            "type": "conversation_part",
            "id": "21874027495",
            "part_type": "default_assignment",
            "body": null,
            "created_at": 1688053574,
            "updated_at": 1688053574,
            "notified_at": 1688053574,
            "assigned_to": { "type": "admin", "id": "4977135" },
            "author": {
              "id": "4977135",
              "type": "bot",
              "name": "Operator",
              "email": "operator+ouy9scuc@intercom.io"
            },
            "attachments": [],
            "external_id": null,
            "redacted": false
          }
        ],
        "total_count": 1
      }
    }
  },
  "links": {},
  "id": "notif_067a1b9c-bb96-7f9b-818f-13f620e4172c",
  "topic": "conversation.user.created",
  "delivery_status": "pending",
  "delivery_attempts": 1,
  "delivered_at": 0,
  "first_sent_at": 1688053575,
  "created_at": 1688053575,
  "self": null
}